### PR TITLE
Linux support

### PR DESCRIPTION
**Pull request by ianaz**
Fixes issue #9 at least for me under Linux. 
Could someone test it on Mac and Windows?

### CODE REVIEW

**Comment by DalderupMaurice** [comment]
Tested it on both Windows + Mac, changes work. Thank you very much! 😄

File: src/createApp.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ import install from "./install";
 
 export default function createApp(answers) {
   const projectName = answers.name;
-  const directory = path.resolve(process.cwd() + "\\" + projectName);
-  const source = path.resolve(__dirname, ("..\\packages\\" + answers.type));
+  const directory = path.resolve(process.cwd() + "/" + projectName);
+  const source = path.resolve(__dirname, ("../packages/" + answers.type));
```
Comment (mhuggins) [inline src/createApp.js]: `path.resolve` can actually take care of slashes for us if we let it:

```js
const directory = path.resolve(process.cwd(), projectName);
const source = path.resolve(__dirname, "..", "packages", answers.type));
```

Comment (mhuggins) [reply src/createApp.js]: cc @DalderupMaurice

Comment (DalderupMaurice) [reply src/createApp.js]: Didn't knew! 
I'll include this in the next PR - the CLI teel needs huge refactoring. I am currently working on it :D

**Pull request merged**